casbin_configuration/model.conf

```diff
@@ -0,0 +1,14 @@
+[request_definition]
```

Comment:
Is there a way to upload this file to s3 with the manifest.yml or write this config in a .py file?